src/main/resources/application.properties

```diff
@@ -2,16 +2,16 @@ server.port=8080
 spring.application.name=devshop_be
 
 #spring.datasource.url=jdbc:h2:mem:testdb;INIT=CREATE SCHEMA IF NOT EXISTS dossier
-spring.datasource.url=jdbc:h2:./databases/devshop;AUTO_SERVER=TRUE;INIT=CREATE SCHEMA IF NOT EXISTS devshop
+spring.datasource.url=jdbc:h2:~/databases/devshop;AUTO_SERVER=TRUE;INIT=CREATE SCHEMA IF NOT EXISTS DEVSHOP
 spring.datasource.driverClassName=org.h2.Driver
 spring.datasource.username=sa
 spring.datasource.password=
 
 spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
 spring.jpa.hibernate.ddl-auto=update
-#spring.jpa.show-sql=true
```

Comment:
kan geen kwaad, maar deze ga je altijd in commentaar hebben staan, tenzij je effectief aan het debuggen bent tegen de database. Elke query wordt dan in de console gepompt